pygmt/datasets/earth_age.py

```diff
@@ -21,16 +21,16 @@ def load_earth_age(resolution="01d", region=None, registration=None):
     So you'll need an internet connection the first time around.
 
     These grids can also be accessed by passing in the file name
-    **@earth_age**\_\ *res*\[_\ *reg*] to any grid plotting/processing
-    function. *res* is the grid resolution (see below), and *reg* is grid
-    registration type (**p** for pixel registration or **g** for gridline
-    registration).
+    **@earth_age**\_\ *res*\[_\ *reg*] to any grid processing function or
+    plotting method. *res* is the grid resolution (see below), and *reg* is
+    grid registration type (**p** for pixel registration or **g** for
```

Comment:
```suggestion
    the grid registration type (**p** for pixel registration or **g** for
```

Reply:
@michaelgrund I am sorry, I overlooked your review. It's already included in commit https://github.com/GenericMappingTools/pygmt/pull/2688/commits/af2e0540e25229ad92e7f1aedca201a3e77e1e97.

I think it also makes sense the make the forumlations a bit more consistent, please see the commits https://github.com/GenericMappingTools/pygmt/pull/2688/commits/af2e0540e25229ad92e7f1aedca201a3e77e1e97 and https://github.com/GenericMappingTools/pygmt/pull/2688/commits/17f6960a8af9df43b0f59669fb0b6c990c2e7874.